### PR TITLE
Add ginkgo to make install-go-tools command

### PR DESCRIPTION
**Pull request by tjons**
# Description

This PR adds `ginkgo` as a installation requirement in the `_output/bin` directory when running `make install-go-tools`. `ginkgo` is a requirement of the test harness, and should be installed similar to other go tools.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -167,6 +167,7 @@ install-go-tools: mod-download ## Download and install Go dependencies
 	go install github.com/cratonica/2goarray
 	go install github.com/golang/mock/mockgen
 	go install github.com/saiskee/gettercheck
+	go install github.com/onsi/ginkgo/v2/ginkgo@v2.17.0
```
Comment (nfuden) [inline Makefile:170]: Can we link this to the go mod please

**Review by lgadban — COMMENTED** [review]
This tripped me up as well, but it already exists in a different target;
see https://github.com/solo-io/gloo/blob/main/Makefile#L214

File: Makefile (hunk 1 above)
Comment (jenshu) [inline Makefile:170]: this is already installed as part of `install-test-tools`

**Comment by tjons** [comment]
Closing as this is part of a different target.

File: Makefile (hunk 1 above)
Comment (tjons) [reply Makefile:170]: 👍 after internal discussion, we will add it here and preserve it in the `install-test-tools` target as CI uses a lighter dependency set.

Comment (tjons) [reply Makefile:170]: 👍

**Review by sam-heilbron — APPROVED** [review]
Alternative would be to invoke the `install-test-tools` as part of the `install-go-tools`

**Pull request merged**